test/test-browsers.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+const fs = require('fs');
+const path = require('path');
+
+/** @type {Record<string, string[]>} */
+const browsers = {
+  desktop: [
+    'chrome',
+    'edge',
+    'firefox',
+    'ie',
+    'opera',
+    'safari',
+  ],
+  mobile: [
+    'chrome_android',
+    'edge_mobile',
+    'firefox_android',
+    'opera_android',
+    'qq_android',
+    'safari_ios',
+    'samsunginternet_android',
+    'uc_android',
+    'uc_chinese_android',
+    'webview_android',
+  ],
+  server: [
+    'nodejs',
+  ],
+  'webextensions-desktop': [
+    'chrome',
+    'edge',
+    'firefox',
+    'opera',
+    'thunderbird',
+  ],
+  'webextensions-mobile': [
+    'firefox_android',
+    'firefox_ios',
```

Comment:
I think only Firefox for Android has webextensions

Reply:
Well, Firefox for iOS hasn’t yet been added to the schema, so it will fail validation regardless.

The reason I already added it and Thunderbird here is so that I don’t have to rebase #2332 and #2333 or this to add them once they’re merged.

Reply:
Firefox for iOS will have WebExtensions eventually; it's on their roadmap for this year as I understand it.

Reply:
As said in #2332 "We can re-consider this [Firefox iOS] if it should become a reality."